chat-plugins/mafia.js

```diff
@@ -1192,6 +1182,67 @@ class MafiaTracker extends Rooms.RoomGame {
 		return false;
 	}
 
+	/**
+	 * @param {User} user
+	 * @param {boolean} hammer
+	 */
+	enableSelfLynch(user, hammer = false) {
+		const from = this.selfEnabled;
+		const to = hammer ? 'hammer' : true;
+		if (from === to) return user.sendTo(this.room, `|error|Selflynching is already set to Self${hammer ? 'hammering' : 'lynching'}.`);
+		if (from) {
+			this.sendRoom(`Self${from === 'hammer' ? 'hammering' : 'lynching'} has been changed to Self${to === 'hammer' ? 'hammering' : 'lynching'}.`, {declare: true});
+		} else {
+			this.sendRoom(`Self${to === 'hammer' ? 'hammering' : 'lynching'} has been enabled.`, {declare: true});
+		}
+		this.selfEnabled = to;
+		this.updatePlayers();
+	}
+	/**
+	 * @param {User} user
+	 */
+	disableSelfLynch(user) {
```

Comment:
Maybe merge these two into `toggleSelfLynch(user, setting)`? Or maybe "toggle" isn't the best word since its asking for a setting (true/false/'hammer')